packages/app/.env

```diff
@@ -1,2 +1,4 @@
-CHAIN_ID=5
+NEXT_PUBLIC_CHAIN_SLUGS=
+NEXT_PUBLIC_CHAIN_STATUS=
+NEXT_PUBLIC_GRAPH_HOST=
```

Comment:
It might simplify things and make things a little more explicit to just put the whole URL here, e.g. `NEXT_PUBLIC_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/holic/example-nft`

And we can make that the default value here in `.env` so folks start off with a clean + working subgraph, with instructions for pointing at their own URL (local or otherwise)